frontend/src/types/index.ts

```diff
@@ -103,3 +98,11 @@ export interface DirectoryData {
     tooling: Tools;
   };
 }
+
+export interface DatabaseModel {
+  id?: string;
```

Comment:
Is `id` really optional?  Seems like it should be mandatory

Reply:
Other than splitting the types, no, not really. I think this is OK for now TBH